README.md

```diff
@@ -37,22 +37,28 @@ In order to use the device in user space, you need to import the provided [udev
 
 Run `yarp-omega3-server` and send RPC commands to `/yarp-omega3-server/rpc:i`.
 
+> please run the `yarpserver` if you did not already before running the `yarp-omega3-server`
+
 Available commands are
-- help
-- stop (completely disengange robot control)
-- quit (close the module)
-- set_position x y z (send a static position)
-- track_position x y z (send a position, to be used in _streaming_ mode)
-- set_force f_x f_y f_z (send a force reference)
+- `help`
+- `stop` (completely disengange robot control)
+- `quit` (close the module)
+- `set_position(x, y, z)` (send a static position)
+- `set_position_move_param(amax, vmax, jerk)` (limit movement parameters)
+- `track_position(x, y, z)` (send a position, to be used in _streaming_ mode)
+- `set_position_track_param(amax, vmax, jerk)` (limit movement parameters)
```

Comment:
Same as above.

Maybe you can add a section after the list of commands to explain the meaning of the three parameters.